src/postman/transformers/__tests__/request.test.ts

```diff
@@ -0,0 +1,351 @@
+                mediaType: 'multipart/test+form-data',
+                schema: {
+                  type: 'object',
+                  properties: {
+                    k1: { type: 'string', description: undefined },
+                    k2: { type: 'string', description: 'd2' },
+                  },
+                },
+              },
+            ],
+          });
+        });
+      });
+
+      describe('media type is not defined', () => {
+        it('returns body containing schema, example and multipart/form-data media type', () => {
+          expect(
+            transformBody(
+              new RequestBody({
+                mode: 'formdata',
+                formdata: [{ key: 'k1', value: 'v1' }],
+              }),
+            ),
+          ).toEqual({
+            contents: [
+              {
+                examples: [{ key: 'default', value: { k1: 'v1' } }],
+                mediaType: 'multipart/form-data',
+                schema: {
+                  type: 'object',
+                  properties: {
+                    k1: { type: 'string', description: undefined },
+                  },
+                },
+              },
+            ],
+          });
+        });
+      });
+    });
+
+    describe('body is not defined', () => {
+      it('returns default empty body', () => {
+        expect(transformBody(new RequestBody({ mode: 'formdata' }))).toEqual({
+          contents: [
+            {
+              examples: [{ key: 'default', value: {} }],
+              mediaType: 'multipart/form-data',
+              schema: { properties: {}, type: 'object' },
+            },
+          ],
+        });
+      });
+
+      it('returns no body', () => {
+        expect(transformBody({ mode: 'formdata' } as RequestBody)).toBeUndefined();
+      });
+    });
+  });
+
+  describe('body is passed in urlencoded mode', () => {
+    describe('body is defined', () => {
+      describe('media type is defined', () => {
+        it('returns body containing schema, example and given media type', () => {
+          expect(
+            transformBody(
+              new RequestBody({
+                mode: 'urlencoded',
+                urlencoded: [
+                  { key: 'k1', value: 'v1' },
+                  { key: 'k2', value: 'v2', description: 'd2' },
+                ],
+              }),
+              'application/test+x-www-form-urlencoded',
+            ),
+          ).toEqual({
+            contents: [
+              {
+                examples: [{ key: 'default', value: { k1: 'v1', k2: 'v2' } }],
+                mediaType: 'application/test+x-www-form-urlencoded',
+                schema: {
+                  type: 'object',
+                  properties: {
+                    k1: { type: 'string', description: undefined },
+                    k2: { type: 'string', description: 'd2' },
+                  },
+                },
+              },
+            ],
+          });
+        });
+      });
```

Comment:
This whole part looks pretty much identical; although it's not critical it would be beneficial to use `describe.each` — this'd have saved some time during the code review.

Reply:
I'm keeping in mind that `describe.each` exists, however, this time it brings no real value IMO. The only one -we-are-sure-it-is-shared part is `expect...toEqual`. The guts may be always anything. Secondly, inside each `describe` there is most of the time only 2-3 inner tests/describes.

I personally don't feel it will help us here.